ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -242,6 +295,16 @@ export const ScenePlayer: React.FC<IScenePlayerProps> = ({
     };
 
     const player = videojs(videoRef.current!, options);
+    var playDurationHandler = window.setInterval(() => {
+      if (trackTime.current) {
+        playDurationRef.current++;
+        if (playDurationRef.current % 10 == 0) {
+          setUpdatePlayDuration(true);
+        } else {
+          setUpdatePlayDuration(false);
+        }
```

Comment:
This is not a great way to be doing the play duration update. Why is it done this way?

Reply:
I needed to separate the logic that handles set database updates from this counter function. Otherwise, the duration count might be off. My knowledge of React is still limited, so there might be a better way to do this.